rust/Cargo.toml

```diff
@@ -35,10 +37,10 @@ lto = true
 # We also need zcash_client_backend and zcash_client_sqlite, which haven't been published
 # with NU5 updates yet.
 [patch.crates-io]
-zcash_client_backend = { git = 'https://github.com/zcash/librustzcash.git', rev='37fc28634e811fdf8db9274a0080cff17c6b6a09' }
-zcash_client_sqlite = { git = 'https://github.com/zcash/librustzcash.git', rev='37fc28634e811fdf8db9274a0080cff17c6b6a09' }
-zcash_primitives = { git = 'https://github.com/zcash/librustzcash.git', rev='37fc28634e811fdf8db9274a0080cff17c6b6a09' }
-zcash_proofs = { git = 'https://github.com/zcash/librustzcash.git', rev='37fc28634e811fdf8db9274a0080cff17c6b6a09' }
+zcash_client_backend = { git = 'https://github.com/zcash/librustzcash.git', rev='ca84123038b64f0f4aa5615f7cf224fbf7ece766' }
+zcash_client_sqlite = { git = 'https://github.com/zcash/librustzcash.git', rev='ca84123038b64f0f4aa5615f7cf224fbf7ece766' }
+zcash_primitives = { git = 'https://github.com/zcash/librustzcash.git', rev='ca84123038b64f0f4aa5615f7cf224fbf7ece766' }
+zcash_proofs = { git = 'https://github.com/zcash/librustzcash.git', rev='ca84123038b64f0f4aa5615f7cf224fbf7ece766' }
```

Comment:
I verified that this corresponds to the merge commit of the Rust migrations (https://github.com/zcash/librustzcash/pull/609). However, it does not include the subsequent change to `WalletMigration0` in https://github.com/zcash/librustzcash/pull/612; this should be pulled in as well so we are switching uniformly to stable migrations:
```suggestion
zcash_client_backend = { git = 'https://github.com/zcash/librustzcash.git', rev='5dceb93ec8aba7d831f6fdacdbbcf3ee3667779d' }
zcash_client_sqlite = { git = 'https://github.com/zcash/librustzcash.git', rev='5dceb93ec8aba7d831f6fdacdbbcf3ee3667779d' }
zcash_primitives = { git = 'https://github.com/zcash/librustzcash.git', rev='5dceb93ec8aba7d831f6fdacdbbcf3ee3667779d' }
zcash_proofs = { git = 'https://github.com/zcash/librustzcash.git', rev='5dceb93ec8aba7d831f6fdacdbbcf3ee3667779d' }
```